### PR TITLE
feat: new user role

### PR DESCRIPTION
**Pull request by liruchen32**
# Issue
[[前端]新增身份的key mapping](https://app.asana.com/0/1203699264568808/1205656532044866/f)

# Notice

# Dependency

### CODE REVIEW

File: packages/core/src/constants/member-role.js [hunk 1]
```diff
@@ -2,6 +2,8 @@ export const MEMBER_ROLE = {
   explorer: 'explorer',
```
Comment (Aylie-Chou) [inline packages/core/src/constants/member-role.js:2]: Could we add some comments to indicate the meaning of `MEMBER_ROLE` mapping ?

Comment (liruchen32) [reply packages/core/src/constants/member-role.js:2]: add

File: packages/redux/src/actions/__test__/mocks/user.js [hunk 2]
```diff
@@ -10,7 +10,13 @@ const mockGetUserData = {
   firstName: 'first',
   lastName: 'last',
   roles: [
-    { id: 1, name: '探索者', name_us: 'explorer', key: MEMBER_ROLE.explorer },
+    {
+      id: 1,
+      name: '探索者',
+      name_us: 'explorer',
+      key: MEMBER_ROLE.explorer,
+      weight: 1,
```
[the remark this answers is not in the document]
Comment (liruchen32) [reply packages/redux/src/actions/__test__/mocks/user.js]: fixed

**Review by Aylie-Chou — APPROVED** [review]
ＬＧＴＭ

**Pull request merged**